### ISSUE TITLE
Fabric toggle control does not support high contrast

### ISSUE THREAD

**Issue by esabherwal** [microsoft/fluentui#12298]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

### Describe the issue:

Currently the toggle does not support high contrast.

#### Actual behavior:

The background of the toggle should be cyan when on (or whatever the user setting is for high contrast). 

#### Expected behavior:

Instead, the background of the toggle is black. 

### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)
High

Products/sites affected: (if applicable)

**Comment by aneeshack4** [comment]
I repro this - upon pressed, it turns cyan blue:
![Uploading image.png…]()
But once you move away, the cyan goes away:
![image](https://user-images.githubusercontent.com/4161791/76664746-a513b300-6542-11ea-89e3-c934503ec004.png)

**Comment by aneeshack4** [comment]
I don't see why though... in `Toggle.styles.ts` the `pill` styles have the right HighContrastSelectors - : https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts#L104 @phkuo any thoughts? I'm not sure what would cause it to be overriden just from clicking somewhere else/focus moving away...

**Comment by phkuo** [comment]
Probably because it's a button and the default HC styles for a button are kicking in.
